### PR TITLE
Native fixes

### PR DESCRIPTION
**Pull request by piggz**
Misc fixes

### CODE REVIEW

File: sparse/lib/systemd/system/wlan-module-load.service [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Load wifi module
+After=local-fs.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=/bin/sleep 5
+ExecStart=/sbin/modprobe bcmdhd firmware_path=/system/vendor/firmware/brcm/fw_bcmdhd_4356a2_pcie.bin nvram_path=/system/vendor/firmware/brcm/nvram_pcie_4356_a2.cal
```
Comment (g7) [inline sparse/lib/systemd/system/wlan-module-load.service]: Wouldn't be better loading up the bcmdhd module via the facility systemd provides (systemd-modules-load, /etc/modules-load.d)? We could get rid of this unit and eventually we'll need to only adjust the startup order. 

What do you think?

**Comment by g7** [comment]
A minor nitpick, otherwise LGTM :tada:

File: sparse/lib/systemd/system/wlan-module-load.service (hunk 1 above)
Comment (piggz) [reply sparse/lib/systemd/system/wlan-module-load.service]: The reason for the sleep 5 hack is.to wait for the droid filesystems to be mounted, including the one which contains the wlan mac addr.  options are to fix the load order before loading the module,  or, store the mac addr on another filesystem. Thoughts?

Comment (g7) [reply sparse/lib/systemd/system/wlan-module-load.service]: Hmm... `After=local-fs.target` should be enough I think (system.mount which mounts /system gets installed in the local-fs target). Perhaps also adding `Requires=local-fs.target` would enforce that even more...

Comment (piggz) [reply sparse/lib/systemd/system/wlan-module-load.service]: Mac addr is on /config/wifi/mac.txt btw

**Comment by piggz** [comment]
I now load wlan using the systemd loader, and it actually works better!  Also added the udev rules fix for block devices, to override the default version.

**Comment by g7** [comment]
Awesome! Everything seems good to me, merging!

**Pull request merged**